.eslintrc.json

```diff
@@ -1,7 +1,7 @@
 {
   "root": true,
   "parserOptions": {
-    "ecmaVersion": 6
+    "ecmaVersion": 2017
```

Comment:
I'm not sure what this is for, but if it's intentional I'll take your word for it.

Reply:
Otherwise eslint is not happy with async keyword and I think async await keeps this code cleaner.